client/src/legacy/LeftAndMain.js

```diff
@@ -460,6 +460,24 @@ $.entwine('ss', function($) {
       // default to first button if none given - simulates browser behaviour
       if(!button) button = this.find('.btn-toolbar :submit:first');
 
+      if ($(button).is("button")) {
+        $(button).append(
+          $(
+            '<div class="btn__loading-icon">' +
+              '<span class="btn__circle btn__circle--1"></span>' +
+              '<span class="btn__circle btn__circle--2"></span>' +
+              '<span class="btn__circle btn__circle--3"></span>' +
+              "</div>"
+          )
+        );
+
+        $(button).css($(button).outerWidth() + "px");
+      }
+
+      // set button to "submitting" state
+      $(button).addClass('btn--loading loading');
+      $(button).prop('disabled', true);
```

Comment:
This used to happen before the conditional block. That should probably remain the case.